### DISCUSSION TITLE
Adding refine to a project

### DISCUSSION THREAD

**Discussion by ghbishal** [refinedev/refine#5454] [Q&A] [answered]
is there any docs or anything that will help me integrate refine to current running project?

**Comment by BatuhanW** [comment]
Hello @ghbishal, we'll add some section to clarify this one. But it shouldn't be different. You just need to install `@refinedev/core` package, and import `Refine` component, children of Refine component can use Refine features.

**Comment by BatuhanW** [comment, marked as answer]
Update: We have a PR waiting to be merged soon, where we add documentation page about using Refine with existing projects: https://github.com/refinedev/refine/pull/5521

Once it's merged, you can find the guide on https://refine.dev/docs/guides-concepts/usage-with-existing-projects
